### PR TITLE
UI For GVI Search

### PR DESCRIPTION
**Pull request by joneubank**
NOTE: This is merging into a new feature branch for GVI that was created from develop. We can collect the GVI work there without affecting release schedule.

This PR also includes the schema updates since those have also been held back from merge into develop.

### CODE REVIEW

File: ui/src/components/search/SearchOptionsDropdown.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        border-radius: 0 0 ${dropdownBorderRadius} ${dropdownBorderRadius};
+      `}
+    >
+      {options.map((option, index) => (
+        <DropdownItem
+          key={`dropdown-item-${option.value}`}
+          title={option.title}
+          details={option.details}
+          value={option.value}
+          searchText="hi"
```
Comment (mistryrn) [inline ui/src/components/search/SearchOptionsDropdown.js:105]: Is `searchText` supposed to get a value from a prop or something?

File: cms/src/schemas/model.js [hunk 2]
```diff
@@ -31,7 +45,7 @@ export const ModelSchema = new mongoose.Schema(
     time_to_split: { type: String, es_indexed: true },
     gender: { type: String, es_indexed: true },
     race: { type: String, es_indexed: true },
-    age_at_diagnosis: { type: Number, es_indexed: true },
+    age_at_diagnosis: { type: Number, ezs_indexed: true },
```
[the remark this answers is not in the document]
Comment (joneubank) [reply cms/src/schemas/model.js:48]: lol typos. thanks for catching this.

File: ui/src/icons/ModelIcon.js [hunk 3]
```diff
@@ -5,6 +5,6 @@ export default ({ fill = '#900000', height = '30px', width = '30px', ...props })
...
...5.25,0,0,0,95,52v-.19a2.5,2.5,0,0,1,5,.23v.15A50.35,50.35,0,0,1,98.06,64,2.5,2.5,0,0,1,95.66,65.77ZM2.5,50.47H2.38A2.51,2.51,0,0,1,0,47.85L2.5,48,0,47.8A50.1,50.1,0,0,1,1.91,36.13a2.5,2.5,0,0,1,4.81,1.38A44.64,44.64,0,0,0,5,48v.1A2.5,2.5,0,0,1,2.5,50.47ZM94.68,36.25a2.5,2.5,0,0,1-2.35-1.65,45.25,45.25,0,0,0-4.83-9.54,2.5,2.5,0,1,1,4.16-2.77A49.59,49.59,0,0,1,97,32.89a2.5,2.5,0,0,1-1.49,3.21A2.55,2.55,0,0,1,94.68,36.25ZM12.75,23a2.51,2.51,0,0,1-2-4.06,50.32,50.32,0,0,1,8.45-8.37,2.5,2.5,0,0,1,3.07,4A45.12,45.12,0,0,0,14.71,22,2.51,2.51,0,0,1,12.75,23ZM76.6,13.11a2.5,2.5,0,0,1-1.4-.43,44.78,44.78,0,0,0-9.51-4.89A2.5,2.5,0,1,1,67.43,3.1,49.94,49.94,0,0,1,78,8.54a2.5,2.5,0,0,1-1.41,4.57ZM37.21,6.72a2.5,2.5,0,0,1-.67-4.91A49.59,49.59,0,0,1,48.29,0a2.5,2.5,0,1,1,.17,5A45,45,0,0,0,37.88,6.62,2.22,2.22,0,0,1,37.21,6.72Z" transform="translate(0 0)"/><path class="cls-1" d="M50,90A40,40,0,1,1,90,50,40.07,40.07,0,0,1,50,90Zm0-75A35,35,0,1,0,85,50,35.06,35.06,0,0,0,50,15Z" transform="translate(0 0)"/><path class="cls-1" d="M57.85,51.11A10.82,10.82,0,1,1,68.66,40.29,10.83,10.83,0,0,1,57.85,51.11Zm0-16.63a5.81,5.81,0,1,0,5.81,5.81A5.82,5.82,0,0,0,57.85,34.48Z" transform="translate(0 0)"/></svg>`,
```
[the remark this answers is not in the document]
Comment (joneubank) [reply ui/src/icons/ModelIcon.js:8]: I added these cause the width/height props were not sizing the svg before I added this. Possible that I implemented something inconsistently, but now I'm concerned our icons are handling this in different ways...

File: ui/src/components/search/SidebarTextSearch.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                      {
+                        op: 'in',
+                        content: {
+                          field: filterField,
+                          value: [selected],
+                        },
+                      },
+                    ],
+                  };
+                  const clearedSqon = SQONUtils.removeSQON(filterField, sqon);
```
[the remark this answers is not in the document]
Comment (joneubank) [reply ui/src/components/search/SidebarTextSearch.js:104]: Yeup! Just like that.

File: ui/src/components/search/SearchOptionsDropdown.js (hunk 1 above)
Comment (joneubank) [reply ui/src/components/search/SearchOptionsDropdown.js:105]: Oops, that should be removed. Something I was working with that didn't get included.

**Pull request merged**